### PR TITLE
Make SequencedCommand serializable and deserializable

### PR DESCRIPTION
**Pull request by frankmcsherry**
This WIP PR moves peek responses back to the coordinator as data, who then communicates along the `mpsc`s to the recipients. The intent is that the prior behavior is preserved, including sending multiple cancellations as if from all outstanding workers, because I don't know how the protocol on the other end of the channel works.

Still to do: CI probably breaks five different ways, but at least we'll see.

### CODE REVIEW

**Comment by frankmcsherry** [comment]
This is blocked on a glitch in `coordtest` where it does not continue to drain the responses from dataflows while awaiting row responses. I believe @mjibson is on it.

**Review by mjibson — APPROVED** [review]

File: src/dataflow/src/server.rs [hunk 1]
```diff
@@ -157,8 +155,8 @@ pub enum SequencedCommand {
         /// The timestamp to advance to.
         advance_to: Timestamp,
     },
-    /// Request that feedback is streamed to the provided channel.
-    EnableFeedback(mpsc::UnboundedSender<WorkerFeedbackWithMeta>),
+    // /// Request that feedback is streamed to the provided channel.
```
Comment (mjibson) [inline src/dataflow/src/server.rs:158]: Remove these

Comment (frankmcsherry) [reply src/dataflow/src/server.rs:158]: Will do. Do you think there are more? (( Did I do this, or a merge artifact? ))

Comment (mjibson) [reply src/dataflow/src/server.rs:158]: I assumed it was leftover from WIP that didn't get cleaned up. I didn't see anything else when I read through everything.

Comment (frankmcsherry) [reply src/dataflow/src/server.rs:158]: 👍

File: src/coord/src/coord.rs [hunk 2]
```diff
@@ -582,6 +590,44 @@ impl Coordinator {
...
+                        }
+                        TailResponse::Complete => {
+                            // TODO: Indicate this explicitly.
+                            self.pending_tails.remove(&sink_id);
+                        }
+                        TailResponse::Dropped => {
+                            // TODO: Could perhaps do this earlier, in response to DROP SINK.
+                            self.pending_tails.remove(&sink_id);
+                        }
+                    }
```
Comment (frankmcsherry) [inline src/coord/src/coord.rs:628]: @benesch I imported a bunch of TODO items from `tail.rs` some of which had your name in them. It seems like, e.g. the sink dropping should be easier to perform here at the coordinator vs back at the tail operator.

**Comment by frankmcsherry** [comment]
There is still some follow-up work, like converting the struct that sends a `TailResponse::Dropped` on drop into something that better encapsulates the protocol (e.g., sending *through* it, rather than a bare shared buffer). Also, potentially renaming `SequencedCommand` to just `dataflow::Command`. Will do in follow-up work.

File: src/coord/src/coord.rs (hunk 2 above)
Comment (benesch) [reply src/coord/src/coord.rs:628]: SGTM!!

**Pull request merged**